setup.py

```diff
@@ -87,21 +83,23 @@ def read(*names, **kwargs):
         'scipy',
         'shapely',
         'xarray',
-        'xlwings >= 0.15.4',
+        'xlwings; platform_system != "Linux"',
         'dataclasses; python_version=="3.6"'
     ],
     extras_require={
         'docs': [
             'm2r',
             'sphinxcontrib-napoleon',
-            'sphinx >= 1.4',
+            'sphinx >= 1.4, < 1.8',
```

Comment:
@Centrus007, turns out a lot of the weird styling that was showing up online was due to backwards incompatibility between Sphinx 1.7 and Sphinx 1.8. I forced the build to be less than 1.8, and now the online docs look much better...

Reply:
See https://pymedphys.com/en/unreleased-docs/index.html now.

Reply:
@SimonBiggs, this was actually a great pick up, mate! I think it was this issue that was causing some less than ideal docstring formatting. Sphinx's parameter auto-documentation is looking a lot better, especially when the parameter type is supplied next to the parameter name. E.g. see [the gamma_shell() docstring in 0.7.x](https://pymedphys.com/en/stable/user/library/gamma.html#pymedphys.gamma.gamma_shell) to the 0.6.0 version.